src/test/scala/cromwell/CallCachingWorkflowSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell
+
+import akka.testkit._
+import cromwell.binding.types.{WdlIntegerType, WdlStringType, WdlArrayType}
+import cromwell.binding.values.{WdlInteger, WdlFile, WdlArray, WdlString}
+import cromwell.CromwellSpec.DockerTest
+import cromwell.util.SampleWdl
+
+import scala.language.postfixOps
+
+class CallCachingWorkflowSpec extends CromwellTestkitSpec("CallCachingWorkflowSpec") {
+  "A workflow which is run twice" should {
+    "use cached calls on the second run" in {
+
+      /** This workflow has two identical calls in it.
+        *
+        * Therefore the first invocation will get ONE call cache hit to itself
+        * The second invocation will get TWO call cache hits.
+        *
+        * Altogether there are THREE call cache hits to the exact same call
+        */
+      runWdlAndAssertOutputs(
+        sampleWdl = SampleWdl.FilePassingWorkflow,
+        eventFilter = EventFilter.info(pattern = s"Call Caching: Cache hit. Using UUID\\(.{8}\\):a\\.*", occurrences = 1),
```

Comment:
@geoffjentry Yeah, the string interpolation isn't used and this string can be extracted
@cjllanwarne It happens once in the first workflow because it actually is running two identical tasks in sequence so the second call invocation gets a cache hit for the first call invocation